src/main/webapp/app/exercises/modeling/assess/modeling-assessment.component.ts

```diff
@@ -72,11 +72,11 @@ export class ModelingAssessmentComponent implements AfterViewInit, OnDestroy, On
             }
             interact('.resizable')
                 .resizable({
-                    edges: { left: false, right: '.draggable-right', bottom: false, top: false },
+                    edges: { left: false, right: this.resizeOptions.initialWidth && '.draggable-right', bottom: this.resizeOptions.verticalResize, top: false },
                     modifiers: [
                         interact.modifiers!.restrictSize({
-                            min: { width: 15, height: 0 },
-                            max: { width: this.resizeOptions.maxWidth ? this.resizeOptions.maxWidth : 2500, height: 2000 },
+                            min: { width: 15, height: 1000 },
+                            max: { width: this.resizeOptions.maxWidth ? this.resizeOptions.maxWidth : 2500, height: 20000 },
```

Comment:
These seem like very magical numbers. Could we maybe have it in a constant instead 🤔 ?